### ISSUE TITLE
Build an Invoice Generator App with NuxtJS, Strapi & Tailwind CSS

### ISSUE THREAD

**Issue by ejirocodes** [strapi/community-content#444]
**My resource**
In this article, we will walk readers through how to build an elegant and functional app for generating invoice that's powered by a Strapi backend

**Outline:**

- Introduction
- Goals
- Prerequisites
- Creating a Strapi Project
- Creating Content Types
- Setting up Roles and Permissions
- Bootstrap a NuxtJS Project
- Adding TailwindCSS to our project
- Building the frontend 
- Connecting to our  Strapi backend
- Generating Invoice
- Conclusion

**My content is**

- [x] A Strapi Tutorial / Guide / How to article
- [ ] About Headless, CMS, javascript, open source, or jamstack
- [ ] Other, specify

**What do you all think? @malgamves, @ShadaW11  **
❤️ I love it!
🚀 I can help you!

**Comment by ShadaW11** [comment]
Hey @ejirocodes interesting topic. However, would you able to use Nextjs instead?

**Comment by ejirocodes** [comment]
Absolutely, I'll update the topic and start working on the article ASAP. Let me know if this sits well with you

**Comment by ShadaW11** [comment]
Hello @ejirocodes thank you for sending the outline. As mentioned in the email and on discord, we have paused the tutorials for now until the release of the V4. I will get back to you in the next couple weeks with some updates.

**Comment by ejirocodes** [comment]
Thanks for the update @ShadaW11

**Comment by ejirocodes** [comment]
@ShadaW11, here is a [link](https://www.dropbox.com/scl/fi/xby10akl8sv76itznnmbz/Build-an-Invoice-Generator-App-with-Next.js-Strapi-Tailwind-CSS.paper?dl=0&rlkey=iuwrxegquh6v4v5b0o4gap2x8) to the article, let me know what you think.

**Comment by ShadaW11** [comment]
Hello @ejirocodes The topic was put on hold as mentioned in my message before. Please do not work on articles that are not approved. I will try my best to find a spot for your article and get back to you.

**Comment by ShadaW11** [comment]
Hello @ejirocodes I sent you an email, kindly get back to me asap.

**Comment by ShadaW11** [comment]
@ejirocodes your article is scheduled for posting, please check your email and fix the 404 issue asap.

**Comment by ejirocodes** [comment]
Will do. Thanks @ShadaW11

**Comment by ejirocodes** [comment]
I've sent the invoice as requested @ShadaW11

**Comment by phuong999** [comment]
I tried to follow this tutorial however this tutorial does not work and is very unclear. I have cloned the code from github and also not working as explained by this article. At the and I achieved what i wanted but this tutorial needs to be updated.
